src/Drawer/Drawer.js

```diff
@@ -166,10 +374,13 @@ class Drawer extends React.Component {
     return (
       <Modal
         BackdropProps={{
+          ref: ref => {
+            this.backdrop = ref != null ? ReactDOM.findDOMNode(ref) : null;
```

Comment:
What about making the `ReactDOM.findDOMNode(` call lazily outside of the ref callback? The ref callback will be called at each render, while we will only need it from time to time.

Reply:
The `ref` callback will _not be called_ on each renderer but only when mounting/unmounting the component. The DOM nodes are needed while swiping, and I want to make the swiping code as fast as possible to reduce lag on mobile devices, so I do the lookup once after mounting.